clang/test/SemaHLSL/BuiltIns/half-float-only-errors2.hlsl

```diff
@@ -0,0 +1,7 @@
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm-only -disable-llvm-passes -verify -DTEST_FUNC=__builtin_elementwise_atan2
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -emit-llvm-only -disable-llvm-passes -verify -DTEST_FUNC=__builtin_elementwise_pow

```

Comment:
remove the https://github.com/llvm/llvm-project/blob/main/clang/test/SemaHLSL/BuiltIns/pow-errors.hlsl file since you are adding this.